### PR TITLE
Adding visualizations for all analyses

### PR DESCRIPTION
**Pull request by twashing**
- Adding visualizations for all analyses
- Setting up a separate repl for figwheel
- Cleaning up scanner workbench

screen shot 2018-09-16 at 5 51 03 pm
screen shot 2018-09-16 at 5 51 36 pm

### CODE REVIEW

**Comment by xiongtx** [comment]
R.e. the scanner workbench, take a look at #25's latest commit. That should make using the scanner much simpler.

File: project.clj [hunk 1]
```diff
@@ -84,13 +84,13 @@
                              [lein-cljsbuild "1.1.7"]
                              [lein-environ "1.1.0"]
                              [lein-figwheel "0.5.16"]
-                             [lein-virgil "0.1.8"]]
+                             #_[lein-virgil "0.1.8"]]
```
Comment (xiongtx) [inline project.clj]: Was this really the cause of slowdown?

File: src/clojure/com/interrupt/ibgateway/component/figwheel.clj [hunk 2]
```diff
@@ -2,7 +2,7 @@
   (:require [mount.core :refer [defstate] :as mount]
             [figwheel-sidecar.repl-api :as ra]))
 
-#_(defstate figwheel
+(defstate figwheel
```
Comment (xiongtx) [inline src/clojure/com/interrupt/ibgateway/component/figwheel.clj:5]: Why do we need both this and `com.interrupt.ibgateway.component.figwheel.figwheel`?

File: src/clojure/com/interrupt/ibgateway/component/figwheel/repl_server.clj [hunk 3]
```diff
@@ -0,0 +1,19 @@
+(ns com.interrupt.ibgateway.component.figwheel.repl-server
+  (:require [environ.core :refer [env]]
+            [mount.core :refer [defstate] :as mount]
+            [clojure.tools.nrepl.server :refer [start-server stop-server] :as nrepl]
+            [cider.nrepl :refer [cider-middleware]]
+            [refactor-nrepl.middleware :refer [wrap-refactor]]))
+
+
+(def host "0.0.0.0")
```
Comment (xiongtx) [inline src/clojure/com/interrupt/ibgateway/component/figwheel/repl_server.clj:9]: Let's make `host` and `env` env vars instead of hard-coding them.

**Comment by xiongtx** [comment]
:clap:  on the :chart:. General question: do we want a number of subplots or to put everything on the same plot?

File: src/clojure/com/interrupt/ibgateway/component/figwheel.clj (hunk 2 above)
Comment (twashing) [reply src/clojure/com/interrupt/ibgateway/component/figwheel.clj:5]: Good question. I'm still trying to have a figwheel service. But that would have to connect to a second repl, if we want both a clojure and clojurescript repl.

File: src/clojure/com/interrupt/ibgateway/component/figwheel/repl_server.clj (hunk 3 above)
Comment (twashing) [reply src/clojure/com/interrupt/ibgateway/component/figwheel/repl_server.clj:9]: Agreed. That was the first thing I tried. The repl server was blowing up on those values. Don't know why.

**Comment by twashing** [comment]
`sma`, `ema` and `bb` are all on the same plot. I found that any more than that, just gets crowded. We could play around with it though. This is just a first pass. For me, it's very valuable to be able to visualize the analyses that the system is calculating.

File: project.clj (hunk 1 above)
Comment (twashing) [reply project.clj]: It's definitely slow regardless. I'll put it back.

**Comment by twashing** [comment]
The more I think about it, the UI should be it's own separate app. So I'll make a refactoring task for that.

**Pull request merged**